plugins/cicd-statistics/api-report.md

```diff
@@ -0,0 +1,182 @@
+## API Report File for "@backstage/plugin-cicd-statistics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { ApiRef } from '@backstage/core-plugin-api';
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-missing-release-tag) "AbortError" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export class AbortError extends Error {}
+
+// Warning: (ae-missing-release-tag) "branchTypes" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const branchTypes: Array<FilterBranchType>;
```

Comment:
Arguably fine to not export this? Or be you have some usage planned :grin:

Reply:
It's silly perhaps, but I was thinking that the *api implementation* might want it... It's not at all necessary though, I'll remove it.